cofmpy/helpers/construct_config.py

```diff
@@ -0,0 +1,357 @@
+        if fmu_from not in fmus:
+            fmus.append(fmu_from)
+        if fmu_to not in fmus:
+            fmus.append(fmu_to)
+
+    return fmus
+
+
+def construct_exported_outputs_list(csvfile_connections):
+    """
+    Construct exported output list from connection list : all sources
+
+    Args:
+        csvfile_connections: connection list file (csv).
+
+    Returns:
+        Dictionary of all exported signals.
+    """
+    exported_outputs = []
+    csvfile_connections.seek(0)
+    connection_itr = csv.DictReader(csvfile_connections)
+    for row in connection_itr:
+        exported_outputs.append(row["from_id"] + "." + row["from_var_name"])
+
+    exported_outputs.sort()
+    return exported_outputs
+
+
+def construct_connection_list(csvfile_connections):
+    """
+    Construct connection list (config format) from connection list (tabular format) :
+        Parse information from connections iterator to construct "connections"
+            section ready to integrate into config file
+
+    Args:
+        csvfile_connections: connection list file (csv).
+
+    Returns:
+        Object list of FMU connections (config file format).
+    """
+
+    connections = []
+    csvfile_connections.seek(0)
+    connection_itr = csv.DictReader(csvfile_connections)
+    for row in connection_itr:
+
+        connections.append(
+            {
+                "source": {
+                    "type": "fmu",
+                    "id": row["from_id"],
+                    "variable": row["from_var_name"],
+                    "unit": "",
+                },
+                "target": {
+                    "type": "fmu",
+                    "id": row["to_id"],
+                    "variable": row["to_var_name"],
+                    "unit": "",
+                },
+            }
+        )
+    return connections
+
+
+def construct_config(connections_path, initializations_path):
+    """
+    Construct global config file for CofmPy :
+        Parse information from connections file to construct "fmus" and "connections"
+            sections
+        parse initializations file to append to "fmus" section
+
+    Args:
+        connections_path (str): Path to the connections file.
+        initializations_path (str): Path to the optional initializations file
+    """
+    console = Console()
+    has_init_file = False
+
+    # Ensure files exists
+    if not os.path.isfile(connections_path):
+        console.print(f"❌ Error: Connection list '{connections_path}' not found.")
+        return
+
+    # check initializations file provided, and exists
+    if initializations_path is not None:
+        has_init_file = True
+        if not os.path.isfile(initializations_path):
+            console.print(
+                f"❌ Error: Initialization list '{initializations_path}' not found."
+            )
+            return
+
+    # Open and read connections_path file
+    with open(connections_path, newline="", encoding="utf-8") as csvfile_connections:
```

Comment:
It's not a big deal, but instead of using directly the text file pointers and do `seek(0)` and `csv.DictReader()` several times in the other functions, it could be faster and simpler to convert them once to dictionaries in the main function `construct_config()` and then propagate dictionaries directly to other functions.